interface-elements-for-web/articles/tree-list/filtering/search-panel.md

```diff
@@ -0,0 +1,96 @@
+---
+title: Search Panel
+author: Anna Kondratova
+---
+# Search Panel
+
+Use the Search Panel to locate data and highlight search result by typing the filter criterion in the search box.
+
+![SearchPanel](../../../images/img17905.png)
+
+## Applying the Search Panel Filter Criterion
+
+Press the ENTER key or click the **Search** button to apply a filter criterion typed in the search panel. Otherwise, a filter is automatically applied in 1.2 seconds.
+
+## Clearing the Search Panel Filter Criterion
+To clear the search panel filter criterion, do one of the following:
+* Press DELETE or BACKSPACE.
+* Click the **Clear** button.
+* Click the clear button in the search box when it is focused and not empty.
+	
+	![EUD_Grid_SearchPanel](../../../images/img25472.png)
+
+## Search syntax
+A search criterion consists of a single word in its simplest form. However, the search panel allows creating composite criteria.
+* **Mask:** criterion 
+	
+	 maria
+	
+	![EUD_Grid_SearchPanelCriterion1](../../../images/img25474.png)
+	
+	**Example description:** selects records that contain the "maria" string in any search column.
+* **Mask:** column:criterion 
+	
+	contact:maria
+	
+	![EUD_Grid_SearchPanelCriterion2](../../../images/img25475.png)
+	
+	End-users can search against a specific column by preceding a search string with the column's caption plus a colon character. Instead of the complete caption, it is possible to use the caption's initial characters to perform a search against the first column whose name starts with the specified substring. To search against a column whose caption contains space characters, specify the column's display caption in quotation marks.
```

Comment:
"End-users"
Shouldn't we change this to just "you" since the readers of End-User Documentation are end-users already?

Reply:
Modified.